### DISCUSSION TITLE
Junicode small caps

### DISCUSSION THREAD

**Discussion by clumperton** [psb1558/Junicode-font#270] [Q&A] [answered]
I have begun using Junicode as a major font in xelatex with fontspec. With
\documentclass[12pt]{article}
\usepackage{fontspec}
\setmainfont{Junicode}
\begin{document}
\textsc{bce}
\end{document} 

I get:
Font shape 'TU/Junicode(0)/m/sc' undefined
using 'TU/Junicode(0)/m/n' instead

I've looked at the Junicode Manual and relevant source files, but I couldn't figure out the solution. I'm using otf fonts. 

I bet there's an easy answer. I'm new to Junicode.

**Comment by Doc73** [comment]
In my system it works very well. I have Junicode installed from CTAN.

```tex
% !TeX TS-program = xelatex
\documentclass[12pt]{article}
\usepackage{fontspec}
\setmainfont{Junicode}
\begin{document}
\textsc{bce}
BCE
\end{document}
```
BTW, the best way is load the font by `\usepackage{junicode}`.

**Comment by psb1558** [comment]
I haven't been able to duplicate this either. The Junicode in CTAN is the latest version, and the package works well. It ought to be in TeXLive 2024 (released a few days ago). I would suggest either upgrading your TeX installation or updating the Junicode package (so that there will be no old version hanging around to confuse things). Then it should work well.

**Comment by clumperton** [comment, marked as answer]
Thank you for your suggestions. I've at least temporarily fixed the problem by using "Junicode-Regular.otf" instead of simply "Junicode". That helped this machine find the font file and then be able to produce the small caps.

**Comment by psb1558** [comment]
Yeah, that sounds good. Glad you found a solution.
